app/screens/paywall/paywall-screen.tsx

```diff
@@ -95,6 +100,18 @@ export function PaywallScreen({ navigation, route }: PaywallScreenProps) {
     })
   }, [])
 
+  const onPurchase = async () => {
+    try {
+      setPurchaseInProgress(true)
+      await purchases.purchaseOffering(subscriptionType)
+    } catch (error) {
+      // TODO: Add crashlytics report here
+      Burnt.alert({ title: "Something went wrong", preset: "error" })
```

Comment:
@planecore what message do you think is suitable here? what are the failure scenarios?

Reply:
Maybe "Sorry... Please try again later"

Reply:
Maybe "Sorry... Please try again later"